src/compose/preprocess.rs

```diff
@@ -242,7 +243,9 @@ impl Preprocessor {
 
         // this code broadly stolen from bevy_render::ShaderProcessor
         let mut lines = shader_str.lines();
-        let mut lines = lines.replace_comments().zip(shader_str.lines()).peekable();
+        let mut lines = lines //.replace_comments()
+            .zip(shader_str.lines())
+            .peekable();
```

Comment:
This is an important change to allow naga to parse the comments. It might incur a performance hit though, so maybe we should feature gate it ? Either way a note as to why comments are stripped may be welcome.

Reply:
Actually no, this change is not necessary, it's only used to read imports, and avoid importing from comments